intelmq/bots/outputs/kafka/output.py

```diff
@@ -0,0 +1,126 @@
+# -*- coding: utf-8 -*-
+
+from json import loads
+from json import dumps
+from json import load
+from collections.abc import Mapping
+from intelmq.lib.utils import load_configuration
+from intelmq.lib.bot import Bot
+try:
+    from confluent_kafka import Producer
+except ImportError:
+    Producer = None
+try:
+    from confluent_kafka import avro
+except ImportError:
+    avro = None
+
+if avro is not None:
```

Comment:
Can this and the two lines above be removed?

```
except ImportError:
    from confluent_kafka.avro import AvroProducer
```

Reply:
I'm just using this to throw better errors in init() for external dependencies. I can remove them if you'd like.